iron-doc-property-styles.html

```diff
@@ -46,7 +46,7 @@
         overflow: hidden;
         text-overflow: ellipsis;
         width: 260px;
-        color: black;
+        color: var(--iron-doc-black, black);
```

Comment:
I think this should be something more generic, like `--iron-doc-viewer-text-color`? Also, could you please document it in a `Styling` section in the header? (you can look at a different element, like `paper-checkbox` on how to do that)